xml/System.Linq/Enumerable.xml

```diff
@@ -1683,7 +1683,11 @@
   
  The <xref:System.Linq.Enumerable.Cast%60%601%28System.Collections.IEnumerable%29> method enables the standard query operators to be invoked on non-generic collections by supplying the necessary type information. For example, <xref:System.Collections.ArrayList> does not implement <xref:System.Collections.Generic.IEnumerable%601>, but by calling <xref:System.Linq.Enumerable.Cast%60%601%28System.Collections.IEnumerable%29> on the <xref:System.Collections.ArrayList> object, the standard query operators can then be used to query the sequence.  
   
- If an element cannot be cast to type `TResult`, this method will throw an exception. To obtain only those elements that can be cast to type `TResult`, use the <xref:System.Linq.Enumerable.OfType%2A> method instead of <xref:System.Linq.Enumerable.Cast%60%601%28System.Collections.IEnumerable%29>.  
+ If an element cannot be converted to type `TResult`, this method will throw an exception.
+
+ The source sequence for this method is `System.Collections.IEnumerable`, which means the elements have the compile time static type of `object`. The only conversion types that are performed by this method are reference conversions and unboxing conversions. The runtime type of the elements in the collection must match the target type, or in the case of value types, the runtime type of elements must be the result of a boxing conversion of the target type. In practice, other conversion types, such as those between different numeric types are not allowed. 
```

Comment:
I've thought a little bit when read about *conversion types* as the sentence before mentions `object` type. So, the word *type* is used in two different meanings in one paragraph. What about:
conversion types --> conversions [or types of conversion]?

And I'd remove *In practice* at the start of the last sentence, as removing it doesn't change a lot, but having it gives impression that in theory things are different.

Reply:
I think I addressed your first comment via @rpetrusha 's comment above. I like the second.